gitops/environments/envs/dev/label-transformer.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: builtin
+kind: LabelTransformer
+metadata:
+  name: <WL_NAME>-dev-labels
+labels:
+  cg-devx.cost-allocation.workload: "<WL_NAME>"
+  cg-devx.cost-allocation.cost-center: "development"
+  cg-devx.metadata.owner: "<WL_NAME>-admins.<WL_NAME>-developers"
+  cg-devx.cost-allocation.environment: "dev"
+fieldSpecs:
+- path: metadata/labels
+  create: true
+  kind: Deployment
```

Comment:
In our sandbox we don't have this line, and I remember that we decided apply labels to all resources, not only to deployments.